packages/webpack/runtime-wrapper-webpack-plugin/test/cases/variables/request-animation-frame/index.js

```diff
@@ -0,0 +1,11 @@
+/*
+// Copyright 2024 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+*/
+expect(requestAnimationFrame).toStrictEqual(expect.any(Function));
+expect(cancelAnimationFrame).toStrictEqual(expect.any(Function));
+expect(requestAnimationFrame).not.toBe(lynx.requestAnimationFrame);
```

Comment:
Same as above, in real Lynx environment, it seems `requestAnimationFrame` should be initialized with `lynx.requestAnimationFrame`.

Reply:
The `requestAnimationFrame` parameter was introduced in Lynx 3.x. To ensure compatibility with older versions of Lynx, you can use `requestAnimationFrame = requestAnimationFrame || lynx.requestAnimationFrame`, allowing you to use `requestAnimationFrame` directly.